### PR TITLE
README.md : Improve structure, fix broken link

### PR DESCRIPTION
**Pull request by lolski**
# Why is this PR needed?

# What does the PR do?
- fix get started link
- improve structure of the 'download and running grakn' section
# Does it break backwards compatibility?

# List of future improvements not on this PR

### CODE REVIEW

**Comment by kasper-piskorski** [comment]
what's with this PR?

**Comment by lolski** [comment]
@kasper-piskorski Fixing README but I haven't managed to act on the review yet.

File: README.md [hunk 1]
```diff
@@ -38,19 +38,16 @@ Grakn’s query language performs distributed [Pregel](https://kowshik.github.io
...
 
 ## Download and Running Grakn Core
-
-To run Grakn Core (which you can download from the [Download Centre](https://grakn.ai/download) or [GitHub Releases](https://github.com/graknlabs/grakn/releases)), you need:
-1. Java 8 (OpenJDK or Oracle Java)
+1. Install Java 8 (OpenJDK or Oracle Java)
 2. If running on Windows version prior to 10, please make sure to have [Visual Studio C++ Runtime for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145) installed
-
-You can visit the [Setup Guide](http://dev.grakn.ai/docs/get-started/setup-guide) to help your installation.
+3. Download Grakn Core from the [Download Centre](https://grakn.ai/download) or [GitHub Releases](https://github.com/graknlabs/grakn/releases)), and follow the [Setup Guide](http://dev.grakn.ai/docs/general/quickstart).
```
[the remark this answers is not in the document]
Comment (lolski) [reply README.md]: PR updated according to your suggestion.

**Comment by lolski** [comment]
closing the PR because the link has been fixed on another PR @sorsaffari

**Pull request closed without merging**